mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -345,12 +345,15 @@ JUGG:
 	WithMoveAnimation:
 		MoveSequence: walk
 	GrantConditionOnDeploy:
+		PauseOnCondition: empdisable || being-captured
 		DeployedCondition: deployed
 		UndeployedCondition: undeployed
 		Facing: 96
 		AllowedTerrainTypes: Clear, Road, DirtRoad, Rough
 		DeploySounds: place2.aud
 		UndeploySounds: clicky1.aud
+		UndeployOnMove: ForceOnly
```

Comment:
I'd set this the other way around. Always for units and ForceOnly for the conyards, on all mods. This is otherwise too easy to trigger and would probably lead to frustrations when playing.

Reply:
I think it should definitely be `ForceOnly` for conyards. I'd even argue that `ForceOnly` should be the default because of the same reason - frustration and anger when you undeploy by accident (could happen if you misclick when shift queuing attack orders for example).

Reply:
I don't remember it ever being a problem with conyards in the original RA2 and why should it? Undeploying is basically the only reason to even select a conyard and then issue and order in the first place. I do remember it being sometimes annoying with artillery in TS because you want to order an attack but misclick (but then again isn't that an issue with all offensive units?).

Regardless, I'm willing to go with the flow here. However, since this is just a simple yaml switch that can be changed with minimal effort at any time in the future, I'll propose instead to leave it at `Always` for all units initially and use future playtests to empirically determine whether and where this is problematic.